src/SolutionTemplate/UnoSolutionTemplate/Wasm/UnoQuickStart.Wasm.csproj

```diff
@@ -47,8 +47,8 @@
 		<PackageReference Include="Microsoft.Extensions.Logging.Filter" Version="1.1.1" />
 		<PackageReference Include="Uno.UI" Version="1.46.196-dev.2440" />
 		<PackageReference Include="Uno.UI.RemoteControl" Version="1.46.196-dev.2440" Condition="'$(Configuration)'=='Debug'" />
-		<PackageReference Include="Uno.Wasm.Bootstrap" Version="1.0.0-dev.302" />
-		<DotNetCliToolReference Include="Uno.Wasm.Bootstrap.Cli" Version="1.0.0-dev.302" />
+		<PackageReference Include="Uno.Wasm.Bootstrap" Version="1.0.0" />
```

Comment:
I'm curious, why?

Reply:
It's basically the same version, but stable :)

Reply:
stability is good :D